deepinv/physics/generator/base.py

```diff
@@ -0,0 +1,113 @@
+import torch
+import torch.nn as nn
+from typing import List
+
+
+class PhysicsGenerator(nn.Module):
+    r"""
+    Base class for parameter generation of physics parameters.
+
+    PhysicsGenerators can be summed to create larger generators (see :meth:`deepinv.physics.generator.PhysicsGenerator.__add__`),
+    or mixed to create a generator that randomly selects (see :meth:`deepinv.physics.generator.GeneratorMixture`).
```

Comment:
Is it possible to document the input args / returns, and to add an example of how the step works in the below example?

Reply:
added!